src/Runtime/OMExternalConstant.inc

```diff
@@ -67,15 +68,18 @@ void omMMapBinaryFile(
   char *tPath = strdup(fname);
   if (!tPath) {
     fprintf(stderr, "Error while strdup");
-    return;
+    return false;
   }
   __a2e_s(tPath);
   fname = tPath;
 #endif
 
   if (constAddr == NULL) {
     perror("Error: null pointer");
```

Comment:
Instead of mixing `fprintf` and `perror`, either use `fprintf` everywhere but add `strerror(errno)` for more informative printout, or use `perror` everywhere.

Reply:
Sure, now using `fprintf` everywhere.